src/include/execution/vm/bytecode_handlers.h

```diff
@@ -292,9 +292,10 @@ VM_OP_HOT void OpPCIGetSlot(terrier::storage::TupleSlot *slot, terrier::executio
 }
 
 VM_OP_HOT void OpPCIGetBool(terrier::execution::sql::BoolVal *out,
-                            terrier::execution::sql::ProjectedColumnsIterator *iter, uint16_t col_idx) {
+                            terrier::execution::sql::ProjectedColumnsIterator *iter,
+                            terrier::storage::col_id_t col_idx) {
   // Read
-  auto *ptr = iter->Get<bool, false>(col_idx, nullptr);
+  auto *ptr = iter->Get<bool, false>(!col_idx, nullptr);
```

Comment:
Modify `terrier::execution::sql::ProjectedColumnsIterator::Get` to accept a `terrier::storage::col_id_t`. Perform the conversion from primitive to strong typedef here.